.github/workflows/redpanda-build.yml

```diff
@@ -24,11 +24,36 @@ jobs:
           run: |
             export DEBIAN_FRONTEND=noninteractive
             apt-get update -y
-            apt-get install -y git sudo pkg-config
+            apt-get install -y git sudo pkg-config ccache
 
         - name: install dependencies
           run: ./install-dependencies.sh
 
+        # there is no analog to ${{ github.workspace }} for the home directory
+        - name: prepare cache variables
+          id: prepare_cache_variables
+          run: |
+            echo ::set-output name=home::$HOME
```

Comment:
I'm using the home directory because that's the default for ccache and I couldn't figure out why if the `CCACHE_DIR` is not empty it gets set to `/dev/shm/redpanda` in `build.sh` - that's why I decided to just point to the default location here.

Reply:
sounds good. wanna push a patch to this branch that tests it?

Reply:
What changes would you suggest? I'm still not sure what's the idea is behind `/dev/shm/redpanda` and decided not to mess with `build.sh`.

Or perhaps you mean to test out the build matrix being incorporated into the cache key?

Reply:
Oh meant testing out the container matrix

Reply:
@dotnwat done, see the latest commit and also this build:
https://github.com/vectorizedio/redpanda/pull/141/checks?check_run_id=1441829844#step:7:7